Apps/WebClient/src/NewClientApp/src/App.vue

```diff
@@ -52,7 +51,8 @@ function currentPathMatches(...paths: string[]): boolean {
         <NotificationCentreComponent />
         <v-main class="position-relative">
             <CommunicationComponent v-if="isCommunicationVisible" />
-            <v-container class="pt-6">
+            <router-view v-if="currentPathMatches(Path.VaccineCard)" />
```

Comment:
struggling to see why you're doing this.

Reply:
is the v-container and issue?

Reply:
Yes, it adds margins around the content, which presents an issue because the page implements its own header.

The same thing happened on the old site, and was solved similarly with the `pageHasCustomLayout` check.  This is just a more streamlined check as the public vaccine card page is the only one where the margin is a problem.

Reply:
@BrianMaki ignore this, no changes needed